projects/angular/src/forms/combobox/_combobox.clarity.scss

```
@@ -323,36 +162,20 @@
 
     &:not(.disabled):hover,
     &:focus-within {
-      @include mixins.css-var(
-        background-color,
-        clr-combobox-hover-background-color,
-        combobox-variables.$clr-combobox-input-background-color,
-        variables.$clr-use-custom-properties
-      );
+      background-color: combobox-variables.$clr-combobox-hover-background-color;
     }
   }
 
   .clr-combobox-trigger {
     &:disabled {
+      // ATM remove opacity - change to fixed color
       color: var(--clr-btn-link-disabled-color, #666);
       opacity: 0.4;
```

`cds-alias-object-interaction-color-disabled` can be used here
```suggestion
      // ATM remove opacity - change to fixed color
      color: var(--clr-btn-link-disabled-color, #666);
      opacity: 0.4;
```

---

added to the caret too